content/en/users/check-in/vos/voms/_index.md

```diff
@@ -285,6 +285,7 @@ voms-proxy-init --voms <name of the vo> --rfc
 
 See for example, using `fedcloud.egi.eu` VO:
 
+<!-- markdownlint-disable line-length -->
 ```{.console}
```

Comment:
@enolfc you didn't touch it right now, but the code fence should more be something like;
```markdown
```sh
echo $PLOP
```

Reply:
I think `{.console}` is a legacy from the restructured doc

Reply:
there is a lot of that:
```
$ git grep -l console content/
content/en/providers/cloud-compute/configuration/_index.md
content/en/providers/cloud-compute/opennebula/_index.md
content/en/providers/cloud-compute/openstack/_index.md
content/en/providers/cloud-compute/validation/_index.md
content/en/users/check-in/vos/voms/_index.md
content/en/users/cloud-compute/auth/_index.md
content/en/users/cloud-compute/ec3/ecas.md
content/en/users/cloud-compute/ec3/htc.md
content/en/users/cloud-compute/faq/_index.md
content/en/users/cloud-compute/openstack/_index.md
content/en/users/cloud-compute/storage/_index.md
content/en/users/cloud-compute/vmi/_index.md
content/en/users/cloud-container-compute/docker.md
content/en/users/data-transfer/WebFTS/_index.md
content/en/users/datahub/_index.md
```

I will fix the check-in ones here

Reply:
Yes basically everything that was imported from restructuredtext have some dirty leftovers that went though the review as mdl wasn't strict enough (and it was lots of content).